build.xml

```diff
@@ -35,13 +35,32 @@
 
   <target name="codestyle" depends="prepare">
     <phpcodesniffer standard="codingstyle.xml">
+      <formatter type="full" usefile="false"/>
       <formatter type="checkstyle" outfile="${builddir}/checkstyle.xml"/>
-      <fileset dir="features">
+      <fileset dir="centreon-build">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="doc">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="GPL_LIB">
+        <include name="**/*.php"/>
+      </fileset>
+      <fileset dir="node_modules">
```

Comment:
the same as for the **vendor** directory

Reply:
yes.
Moreover, this folder doesn't contain php files on which PSR2 coding-style may be applied